### ISSUE TITLE
[NG] Want control over homepage text

### ISSUE THREAD

**Issue by JenMysoc** [mysociety/pombola#1812]
SYE have written to us and asked how difficult it would be to have control over the text for the homepage. I believe this is on our roadmap for the future - however since this is a specific request I've ticketed it for discussion.

**Comment by JenMysoc** [comment]
'Yemi has said the following:

government accountability, especially the national assembly, is a big focus for us over the next 4 years.

is it possible to give a time frame for when we would have more control of the homepage? it's been a pending issue for almost 2 years.

the alternative is for us to create a 'dummy' homepage that we can control with links to the mysociety back-end.

we've gotten several complaints about the site and we are trying to get funding for this part of our work so it's been quite frustrating.

**Comment by tmtmtmtm** [comment]
@JenMysoc do you know what degree of control would be a sufficient first step here? (i.e. — not the all singing, all dancing version — the minimal version that will still add value for them). Is this a new section they'd like to add to what is already on the homepage? Or the ability to change one or more of the parts that currently exist?

**Comment by JenMysoc** [comment]
They wanted to be able to edit the text on the homepage and ideally they'd
like to be able to add html in the future. I think we can get around that
by them having a more modifiable homepage in the future but not necessarily
allowing changes to html that could interact badly with the core code.

**Comment by tmtmtmtm** [comment]
There isn't really a lot of text on the homepage at the minute:

![screen shot 2015-11-13 at 15 58 46](https://cloud.githubusercontent.com/assets/57483/11150329/7244b01c-8a1f-11e5-96a4-7595588b5b01.png)

You mean they want to be able to change the "Shine your eye is an SMS and web platform that facilitates engagement with National Assembly members and other elected officials" part?

**Comment by JenMysoc** [comment]
Yeah they definitely wasn't to be able to change that. There should be a
separate ticket for making the twitter box smaller too. I think nthey want
to add some text in but I don't know what that text would be.

**Comment by paullenz** [comment]
So there are a couple of things here (which broadly apply across all
instances):

1) The ability to edit/manage the nav bar (NB there might be a separate
ticket for this

2) For there to be additional text fields on the homepage that can be
edited - the redesigns for pa.org.za and mzalendo.com have, to an extent
negated the need in their cases as the editorial blog content is much more
prominent on the homepages and achieves this by proxy

3) And yet, the ability for SYE in particular to be able to edit the text
that you cite

**Comment by tmtmtmtm** [comment]
https://github.com/mysociety/pombola/issues/1785 is the ticket for the twitter box. 

I don't remember seeing one for the nav bar, and a quick search doesn't turn anything up.

**Comment by paullenz** [comment]
Hmm - good point - possibly it was never actually created (or was created
with non-obvious terms)

The hunt revealed this ticket to me though -
https://github.com/mysociety/pombola/issues/630 - which I am closing...

**Comment by JenMysoc** [comment]
As this is a ticket specifically for EiE in Nigeria: @tmtmtmtm this is what 'Yemi has suggested they want to do with the homepage http://www.eie.ng/syedemo/ They want to know what needs to happen to make the links work and make it so they can have this and control the content on it.

**Comment by paullenz** [comment]
See also ticket #1298

**Comment by JenMysoc** [comment]
See also #1910
